chatgpt_wrapper/chatgpt.py

```diff
@@ -88,48 +118,71 @@ def _set_logging(self, debug_log):
         return logger
 
     def _start_browser(self):
-        self.page.goto("https://chat.openai.com/")
+        self.log.info("Loading ChatGPT webpage...")
+        ChatGPT.page.goto("https://chat.openai.com/")
+        try:
+            ChatGPT.page.wait_for_url("/chat")
+        except Exception:
+            errors=ChatGPT.page.query_selector_all("div.cf-error-details")
+            if len(errors)>0 and 'Access denied' in errors[0].inner_html():
+                self.log.error("Got Access denied. Self-disabling for 10 minutes. ")
```

Comment:
Not loving the hard-code self disabling thing, what is the purpose of this code? Seems as likely to accidentally lock a user out as to provide benefit.

Reply:
When `OpenAI` gives us an `Access denied` page, I'm afraid that continue to access (frequently) would prolong the denying time. 

This might need further investigating and if I'm wrong please tell me. 

(I don't have ChatGPT Plus)